tests/chainer_tests/training_tests/extensions_tests/test_plot_report.py

```diff
@@ -17,14 +17,16 @@ def test_available(self):
         with warnings.catch_warnings(record=True) as w:
             self.assertEqual(extensions.PlotReport.available(), available)
 
-        # It shows warning only when matplotlib.pyplot is not available
+        # It shows warning only when matplotlib is not available
         if available:
             self.assertEqual(len(w), 0)
         else:
             self.assertEqual(len(w), 1)
 
+    # In Python 2 the above test does not raise UserWarning,
```

Comment:
`available()` raises `UserWaning` in the first time, but it does not produce it in the second time, and therefore `self.assertEqual(len(w), 1)` always fails. As unittest does not guarantee the order of calling test methods, in python2 the above test method fails